cli/compose/schema/data/config_schema_v3.2.json

```diff
@@ -249,7 +248,6 @@
                   "source": {"type": "string"},
                   "target": {"type": "string"},
                   "read_only": {"type": "boolean"},
-                  "consistency": {"type": "string"},
```

Comment:
Looks like the consistency field is in the 3.3 format in docker compose; https://github.com/docker/compose/commit/fa77856c8641c076d8d3652bbe354a7cebfab172

the example in the docker-compose reference shows `version: '3'`, but no mention it was added in version `x.y` https://docs.docker.com/compose/compose-file/#caching-options-for-volume-mounts-docker-for-mac